metron-interface/metron-rest/src/test/java/org/apache/metron/rest/controller/KafkaControllerIntegrationTest.java

```
@@ -148,6 +150,15 @@ public void stop() {
 
   @Before
   public void setup() throws Exception {
+    runner = new ComponentRunner.Builder()
+            .withComponent("kafka", kafkaWithZKComponent)
+            .withCustomShutdownOrder(new String[]{"kafka"})
+            .build();
+    try {
+      runner.start();
+    } catch (UnableToStartException e) {
+      e.printStackTrace();
```

I'd be ok with that.  This will just mask an error until it fails later less explicably.